cider/featurizer.py

```diff
@@ -288,9 +289,7 @@ def write_bc(index: Any, iterator: Any) -> Any:
                                             ('reporting' not in col) or (col == 'reporting__number_of_records')])
         cdr_features = cdr_features.toDF(*[c if c == 'name' else 'cdr_' + c for c in cdr_features.columns])
         save_df(cdr_features, self.outputs_path / 'datasets' / 'bandicoot_features' / 'all.csv')
-        self.features['cdr'] = read_csv(
-            self.spark, self.outputs_path / 'datasets' / 'bandicoot_features' / 'all.csv', header=True, inferSchema=True
-        )
+        self.features['cdr'] = cdr_features
```

Comment:
did this in a few places where we were writing out and then reading back. Assumes that the original dataframe we wrote out has the columns the way we want them, which seemed accurate from doing manual testing.

Reply:
@emilylaiken -- do you remember if there was a reason for this write-then-read idiom?

Reply:
This seems fine. iirc the only reason to read back in is something related to modularity (i.e. if there's a crash, you can always start by reading in the the features). That made more sense back when it was research code; I think here it's fine to assume one successful run from start to finish.